barretenberg/cpp/src/barretenberg/stdlib/recursion/honk/verifier/goblin_verifier.test.cpp

```diff
@@ -10,27 +10,43 @@
 namespace proof_system::plonk::stdlib::recursion::honk {
 
 /**
- * @brief Test suite for recursive verification of conventional Ultra Honk proofs
+ * @brief Test suite for recursive verification of Goblin Ultra Honk proofs
  * @details The recursive verification circuit is arithmetized in two different ways: 1) using the conventional Ultra
  * arithmetization (UltraCircuitBuilder), or 2) a Goblin-style Ultra arithmetization (GoblinUltraCircuitBuilder).
  *
  * @tparam Builder Circuit builder for the recursive verifier circuit
  */
 template <typename BuilderType> class GoblinRecursiveVerifierTest : public testing::Test {
 
-    // Define types relevant for inner circuit
-    using Flavor = ::proof_system::honk::flavor::GoblinUltra;
-    using InnerComposer = ::proof_system::honk::UltraComposer_<Flavor>;
+    // Define types relevant for testing
+    using UltraFlavor = ::proof_system::honk::flavor::Ultra;
+    using GoblinUltraFlavor = ::proof_system::honk::flavor::GoblinUltra;
+    using UltraComposer = ::proof_system::honk::UltraComposer_<UltraFlavor>;
+    using GoblinUltraComposer = ::proof_system::honk::UltraComposer_<GoblinUltraFlavor>;
+
+    using InnerFlavor = GoblinUltraFlavor;
+    using InnerComposer = GoblinUltraComposer;
     using InnerBuilder = typename InnerComposer::CircuitBuilder;
-    using NativeVerifier = ::proof_system::honk::UltraVerifier_<::proof_system::honk::flavor::Ultra>;
     using InnerCurve = bn254<InnerBuilder>;
+    using Commitment = InnerFlavor::Commitment;
```

Comment:
I guess for consistency this should be InnerCommitment and InnerFF but possibly that's too verbose

Reply:
made that change